app/code/Magento/Checkout/view/frontend/web/js/view/billing-address.js

```diff
@@ -38,6 +38,7 @@ function (
     'use strict';
 
     var lastSelectedBillingAddress = null,
+        toCheck = true,
```

Comment:
Hi @webkul-ratnesh. Thanks for the collaboration. What the reason to add additional variable `toCheck`. As I see we can have the same result using only `window.isbothAddressSame` variable

Reply:
yes actually there isn't any need now, i will remove this extra variable.